README.md

```diff
@@ -7,6 +7,48 @@ In addition, a persistent instance will be available on GCP for non-technical us
+    - `postgres_db_read` - the parallel read Postgres instance that is written to simultaneously to the write instance, in order to optimise the concurrent processing of read operations without being detrimental to the performance of write operations.
+- By default, the container port mappings are: 
+    - The `gc_registry` FastAPI app - `8000`
+    - The `eventstore` instance GUI - `2113`
+    - The Postgres write intance - `5436`
+    - The Postgres read instance - `5438` 
+- Documentation comes in two flavours:
+    - `localhost:8000/docs` - Swagger-style schema and endpoint descriptions that allows basic interaction with API
+    - `localhost:8000/redoc` - alternative, non-interactive documentation from Redocly that is more human-readable than Swagger
+- This repository uses a [makefile](Makefile) for quick access to convenience functions that are common across both users and developers. For instance, `make dev` will quickly call Docker compose to load up the container cluster. 
+- On first build, the database containers will not contain the schemas for the entities. We have used `alembic` to manage the schema migrations and for ease of setting up - use `make db.update` to load in the most recent schema to the database.
+    - If during development you make any changes to the `sqlmodel` entity definitions, you can quickly reflect that change in the database by running `make db.revision "NAME={name_your_revision}"`, with a suitably descriptive name for the intended changes in place of `name_your_revision`.
+    - This will create a new migration file in `gc_registry/core/alembic/versions`, and we recommend double checking the contents of this auto-generated file before committing it to the database with `make db.update`.
+    - The database can be fully reset using `make db.reset`; this will clear all entities in both database instances and reset the schema to the most recent iteration.  
+- Initially, the database instances will contain no elements. To get started quickly, we recommend seeding the database with some example User, Account, Device, and GC Bundle entities courtesy of Elexon (the UK aggregator of electricity system data) with the command `make db.seed`.
+
+### Interfacing with the Registry
+
+Each entity has the four basic crud operation endpoints accessible through the lower case singular name of the entity as the router and the operation or ID as the suffix. For example, either through Postman to `localhost:8000` or through a notebook with `client = httpx.Client("localhost:8000)` 
+- `GET(account/1)` will return the account with ID 1, which in this case would be first account created on the registry as the integer ID primary keys are unique and serial across each entity.
+- Creating an entity is possible like so:
+    ```python
+    from gc_registry.account.schemas import AccountBase
+    account = AccountBase(account_name="Example Account", user_ids=[1], account_whitelist=[1])
```

Comment:
Maybe remove use of passing data use `AccountBase` schema as people might not be familiar with approach